gazelle_haskell_modules/lang.go

```diff
@@ -225,6 +227,13 @@ func fixModulesList(r *rule.Rule, ruleNameSet map[string]bool) {
 	fixModulesLists(r, ruleNameSet, "modules", stripColon)
 }
 
+func fixHiddenModulesList(r *rule.Rule, ruleNameSet map[string]bool) {
+	ruleName := r.Name()
+	// TODO: use something better?
+	addPackageName := func(module string) string { return fmt.Sprintf(":%s.%s", ruleName, module) }
```

Comment:
IIUC, you want to drop the colon from the output.

Reply:
What do you mean by "output"? I'm adding this here so it matches the format in `ruleNameSet`. It seems a bit brittle though - is there some way to make it better?

Reply:
I stripped the colon in 5496a59.

Reply:
I'm not really sure why it works both with and without the colon (this code is out of cache right now :sweat_smile: ) but I trust your judgement :)

Reply:
A first short answer is that ruleNameSet doesn't include the leading `:`, so it needs to be removed. Though note that the code has evolved since this conversation. Happy to engage with more details if needed.